apps/emqx_dashboard/src/emqx_dashboard_swagger.erl

```diff
@@ -138,7 +138,14 @@ fields(limit) ->
     Meta = #{in => query, desc => Desc, default => ?DEFAULT_ROW, example => 50},
     [{limit, hoconsc:mk(range(1, ?MAX_ROW_LIMIT), Meta)}];
 fields(count) ->
-    Meta = #{desc => <<"Results count.">>, required => true},
+    Desc = <<
+        "Total count.<br/>"
+        "Note: This value returns 0 when there are no records or when a "
+        "fuzzy searching is performed.<br/>"
+        "Because it's very expensive to get the total count in a fuzzy "
+        "searching."
```

Comment:
```suggestion
        "Note: This value returns 0 when there are no records or when a "
        "fuzzy search is performed.<br/>"
        "Because it's very expensive to get the total count in a fuzzy "
        "search."
```